server/testconnection/pipeline.go

```diff
@@ -0,0 +1,86 @@
+package testconnection
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/google/uuid"
+	"github.com/kubeshop/tracetest/server/datastore"
+	"github.com/kubeshop/tracetest/server/http/middleware"
+	"github.com/kubeshop/tracetest/server/model"
+	"github.com/kubeshop/tracetest/server/pkg/pipeline"
+	"github.com/kubeshop/tracetest/server/tracedb"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+type Job struct {
+	Headers    map[string]string      `json:"headers"`
```

Comment:
are the headers used for anything other than passing the tenant ID? if so, it might be simpler to just have a `TenantID string`  field

Reply:
I wanted to keep using the same approach we have for the rest, but yeah we can just change it for a new field, let me check..